Unix/http/httpauth.c

```diff
@@ -1605,83 +1612,70 @@ void Deauthorize(_In_ Http_SR_SocketData * handler)
     handler->authInfo.gid = -1;
 }
 
-static int _GeneratePipeFile(char *file)
+static MI_Result _ServerAuthenticateCallback(PamCheckUserResp *msg)
 {
-    char rString[32];
+    MI_Result r;
+    char *auth_response = NULL;
+    int response_len = 0;
 
-    GenerateRandomString(rString, sizeof(rString)); 
+    Http_SR_SocketData *handler = (Http_SR_SocketData*)(uintptr_t)msg->handle;
+    HttpHeaders *headers = &handler->recvHeaders;
 
-    Strlcpy(file, OMI_GetPath(ID_SYSCONFDIR), PAL_MAX_PATH_SIZE);
-    Strlcat(file, "/sockets/", PAL_MAX_PATH_SIZE);
-    Strlcat(file, rString, PAL_MAX_PATH_SIZE);
+    // Resume selector processing of this socket
+    handler->handler.mask |= SELECTOR_READ;
+    Selector_Wakeup(handler->http->selector, MI_TRUE);
```

Comment:
need to do this AFTER you set the auth members on the handler otherwise the pam authorization may happen again